features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/AppsAndKeys/ViewKeys.jsx

```diff
@@ -291,6 +292,10 @@ class ViewKeys extends React.Component {
                     this.setState({ notFound: true });
                 }
                 this.setState({ isUpdating: false });
+                const { response } = error;
+                if (response.body) {
```

Comment:
shall we check the response value as well
```
 if (response && response.body) {
```

otherwise, if `response` is undefined this line will fail

Reply:
Fixed with: https://github.com/wso2/carbon-apimgt/pull/9176/commits/3d152a9f1b35a4528529dfe91a884c75eebf1202